### DISCUSSION TITLE
Richiesta

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#405] [Richieste di aiuto] [answered]
Buongiorno,sarebbe possibile avere una guida dell'app GiuaReg v2.0 in modo da capire semplicemente come funziona e come è strutturata?Grazie

**Comment by trinko** [comment, marked as answer]
Non c'è, perché l'app rimanda semplicemente al sito web del registro, senza modifiche. 
Serve solo per poter memorizzare le credenziali di accesso, senza bisogno di inserirle ogni volta.
